src/psyclone/tests/domain/lfric/transformations/lfric_extract_test.py

```diff
@@ -34,6 +34,7 @@
 # Author I. Kavcic, Met Office
 # Modified by A. R. Porter and R, W, Ford, STFC Daresbury Lab
 # Modified by J. Henrichs, Bureau of Meteorology
+# Modified by L. Turner, Met Office
```

Comment:
Not your error, but would you be kind to swap commas for full stops in Rupert's initials in the line 35 ("R. W. Ford")? Thank you :)